src/main/integrations/electron-breadcrumbs.ts

```diff
@@ -183,14 +183,10 @@ export class ElectronBreadcrumbs implements Integration {
         };
 
         if (id) {
-          const state = getRendererProperties(id);
+          breadcrumb.data = { ...getRendererProperties(id) };
```

Comment:
`getRendererProperties` could return `null`

Reply:
Unless I'm mistaken, `getRendererProperties` should only ever return some object or `undefined`, which this change should handle. Am I overlooking something?

Reply:
Oh interesting, I didn't think `...null` was valid JS 🤔 Apparently it is?

Reply:
Yeah this should be fine to cover the case here.